frontend/public/components/impersonate-notifier.jsx

```diff
@@ -21,23 +21,27 @@ export const ImpersonateNotifier = connect((state) => ({ impersonate: getImperso
     : impersonate.kind;
   const impersonateName = impersonate.name;
   return (
-    <div className="co-global-notification">
-      <div className="co-global-notification__content">
-        <p className="co-global-notification__text">
-          <span className="text-uppercase co-global-notification__impersonate-kind">
-            {t('public~Impersonating {{kind}}', {
-              kind: kindTranslated,
-            })}
-          </span>{' '}
+    <Banner color="blue">
```

Comment:
could possibly use `status='info'` prop here for semantics, but that would change the color from blue to purple

Reply:
fair point, it is slightly ambiguous when to use it